bootstrap/main.go

```diff
@@ -193,3 +210,44 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func getAssistedHTTPClient(config assistedinstaller.ServiceConfig, c client.Client) (*http.Client, error) {
```

Comment:
I think this should go in [this](https://github.com/openshift-assisted/cluster-api-agent/tree/master/assistedinstaller) package